### ISSUE TITLE
Mobilenet_emb training configuration

### ISSUE THREAD

**Issue by assaf86** [leondgarse/Keras_insightface#50]
Hi @leondgarse 

Can you please share the training configuration of the **Mobilenet_emb** model you shared?
https://drive.google.com/file/d/1i0B6Hy1clGgfeOYtUXVPNveDEe2DTIBa/view

(Scheduler parameters, loss, # of epochs, etc'...)

Thanks in advance!

**Comment by leondgarse** [comment]
- By "early training test", I mean it's the script I used last year.
- Yes, you can use it for reproducing. All those scripts I shared there are the best ones I tested from many others.
- `emore` means `MS-1M (MS-celeb)` itself, with totally `85741` classes and `5822653` images. I think it's now called [ms1m-arcface](https://github.com/deepinsight/insightface/tree/master/recognition/_datasets_#ms1m-arcface-85k-ids58m-images-57). Besides, `MS1MV3` means `MS1M-RetinaFace`, and it usually performs better than `emore` in my tests. Other prepare process is explained in [README beforehand-data-prepare](https://github.com/leondgarse/Keras_insightface#beforehand-data-prepare).
